e2e/steps_file.js

```diff
@@ -141,6 +141,11 @@ module.exports = function () {
       }
     },
 
+    seeInTabText (pathToField) {
```

Comment:
`seeRowInTab` perhaps?

Reply:
maybe `seeTextInTab` seems more correct to me. Technically, we are just looking for text. Let me know what you think

Reply:
changed name to `seeTextInTab`